app/views/api/v2/realms/main.json.rabl

```diff
@@ -1,3 +1,5 @@
 object @realm
 extends "api/v2/realms/base"
+extends "api/v2/smart_proxies/children_nodes"
+
 attributes :name, :realm_proxy_id, :realm_type, :created_at, :updated_at
```

Comment:
`realm_proxy_id` and `_name` should be replaced by the smart_proxy/children_nodes too I think, it can be generated easily.

Ditto in other views, e.g. host has puppet_proxy_id/puppet_ca_proxy_name which can be replaced.